frontend/packages/console-shared/src/components/status/status.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import * as History from 'history';
+import { Link } from 'react-router-dom';
+import { Popover } from '@patternfly/react-core';
+import { Icon, Button } from 'patternfly-react';
+
+type StatusProps = {
+  icon?: string;
+  children: React.ReactNode;
+};
+
+export const Status = ({ icon, children }: StatusProps) => (
+  <>
+    {icon && <Icon type="pf" name={icon} className="kubevirt-status__icon" />} {children}
+  </>
+);
+
+type PopoverStatusProps = StatusProps & {
+  header: React.ReactNode;
+};
+
+export const PopoverStatus = ({ icon, header, children }: PopoverStatusProps) => (
+  <Popover position="right" headerContent={header} bodyContent={children}>
+    <span className="kubevirt-status__popover">
```

Comment:
We should not use non console classnames in shared components.

Reply:
Yep, I forgot to remove/change this.